utils/rowgroup/rowstorage.h

```diff
@@ -161,6 +164,9 @@ class RowAggStorage
    */
   inline void rowHashToIdx(uint64_t h, uint32_t& info, size_t& idx, const Data* curData) const
   {
+    // An addition from the original robin hood HM.
+    h *= fCurData->hashMultiplier_;
+    h ^= h >> 33U;
```

Comment:
Do they have some explanation on "h >> 33U" why do they do it?

Reply:
Picked up by robin hood as well.